.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: node_js
+
+node_js:
+  - "6"
+
+sudo: enabled
+
+before_install:
+  - pip install psutil
+  - pip install requests
+
+matrix:
+  include:
+    - os: linux
+      dist: trusty
+      sudo: enabled
```

Comment:
duplicated declaration of "sudo: enabled" ?

Reply:
Oops, duplicated.